src/_routing/_components/RootStackNavigator.tsx

```diff
@@ -10,10 +11,13 @@ export type TRootParams = Record<TRootRoutes, undefined>;
 const RootStack = createNativeStackNavigator<TRootParams>();
 
 export const RootStackNavigator = () => {
+  const { user } = useAuth0();
```

Comment:
is de user hier getyped?

Reply:
Good point... Er zijn atm geen types voor de [react-native-auth0](https://www.npmjs.com/package/@types/react-native-auth0) package, want die zitten een versie achter. Dus de types package komt niet overeen met de werkelijkheid. 

Dus we kunnen 2 dingen doen imo:
- Zelf typen (en maintainen)
- Ermee leven dat het niet getyped is en opvolgen wanneer de types package geüpdatet is

Ik verkies het tweede aangezien die Auth0 `user` nooit echt gebruikt gaat worden voor user data (die komt later van de UiTPAS API)

Reply:
akkoord!